lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -499,8 +503,12 @@ export abstract class ClientApplication {
 
         authRequest.authenticationScheme = AuthenticationScheme.BEARER;
 
-        // Set requested claims hash if claims were requested
-        if (authRequest.claims && !StringUtils.isEmpty(authRequest.claims)) {
+        // // Set requested claims hash if claims-based caching is enabled and claims were requested
+        if (
```

Comment:
Consider adding tests for both client_credentials and OBO, as both of these methods look into the cache on their own (not via acquire token silent).

Reply:
Yep, I opened this up for review before adding the node tests, sorry about that just trying to get ahead on the reviews browser-side.

Reply:
Punting Client Credentials and OBO tests and bug fixes to a different PR over the next couple of weeks so we can get these changes merged for PCA before our next release.